src/arrayMethodSort.js

```diff
@@ -4,9 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = comparingElements) {
+    for (let i = 0; i < this.length; i++) {
+      for (let k = i + 1; k < this.length; k++) {
+        if (compareFunction(this[i], this[k]) > 0) {
+          const tempElem = this[k];
+
+          this[k] = this[i];
+          this[i] = tempElem;
+        }
+      }
+    }
+
+    return this;
   };
 }
 
+const comparingElements = (a, b) => {
+  if (String(a) > String(b)) {
+    return 1;
+  }
+
+  if (String(a) < String(b)) {
+    return -1;
+  }
+
+  return 0;
+};
+
```

Comment:
It's a good thing that you have decided to do like this, but there is one much faster way. You could do like this 
`function (compareFunction = (a, b) => `${a}` > `${b}`) {`